lib/bots/syncbot.ts

```diff
@@ -94,9 +110,9 @@ export class SyncBot extends ProcBot {
 					}
 					return emitResponse;
 				})
-				// Then report that we have passed the message on
+				// Once we reach the end of our actions we don't actually care about the response and should void it away
 				.then(() => {
```

Comment:
Should get rid of the `then` if it has no body, and shouldn't be returning the `emitResponse` from the previous promise. Currently it's confusing as it looks like something was going to be done with the body, but was forgotten.